opm/simulators/flow/equil/InitStateEquil.cpp

```diff
@@ -22,15 +22,14 @@
 */
 
 #include <config.h>
-#include <ebos/equil/initstateequil.hh>
-#include <ebos/equil/initstateequil_impl.hh>
+#include <opm/simulators/flow/equil/InitStateEquil_impl.hpp>
```

Comment:
_impl.hpp files includes the .hpp file. Since it happens more than one place, it looks the including of the original .hh or .hpp file is intentional. I do not have an opinion about this, maybe @bska can comment on this.

Reply:
It's a noop as such.The header guard is defined at this point. It's an incomplete attempt at what I do properly in https://github.com/OPM/opm-simulators/pull/5227 (though not for this class).

Reply:
Uhm, this is a different pattern. You have a .hpp, a _impl.hpp and a .cpp file. The cpp file instances for CpGrid. So in the regular simulators we just include the .hpp file and the instance is in the library. For simulators using other grids, we instead include the _impl.hpp file and thus the instance for those grid types are compiled inline in the simulator object.